packages/dds/matrix/src/test/time/.mocharc.cjs

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run memory-profiling tests
+ */
+
+module.exports = {
+	"exit": true,
+	"fgrep": ["@Benchmark", "@ExecutionTime"],
+	"node-option": ["expose-gc", "gc-global", "unhandled-rejections=strict"], // without leading "--"
+	"recursive": true,
+	"reporter": "@fluid-tools/benchmark/dist/MochaReporter.js",
+	"reporterOptions": ["reportDir=.memoryTestsOutput/"],
```

Comment:
Execution time benchmarks use a different path here, look at the `packages/test/test-end-to-end-tests/src/test/benchmark/.mocharc.time.cjs` file. I just realized that in most places we haven't been following the pattern of having a mocha config file for execution time tests, but I like that we're doing it here.

Reply:
Updated